docs/src/_reference/plugin-definition-syntax.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Plugin Definition Syntax
+description: YAML syntax for defining plugins for Meltano Hub.
+layout: doc
+weight: 3
+---
+
+This document describes the syntax for defining plugins contributed to Meltano Hub.
+
+## `name`
+
+Required. The name of the plugin. The tuple `(name, variant)` must be unique.
+
+```yaml
+name: airflow
+```
+
+## `variant`
+
+Required. The name of the variant. New variants of existing plugins usually come form forks or re-implementations.
+
+```yaml
+variant: apache
+```
+
+## `namespace`
+
+Required. This is used to configure multiple plugins that are meant to work together.
+
+```yaml
+namespace: airflow
+```
+
+## `label`
+
+A human-readable label for the plugin.
+
+```yaml
+label: Airflow
+```
+
+## `docs`
+
+The URL of the plugin's documentation.
+
+```yaml
+docs: https://docs.meltano.com/guide/orchestration
+```
+
+## `repo`
+
+The URL of the plugin's repository (in GitHub, GitLab, etc.).
+
+```yaml
+repo: https://github.com/apache/airflow
+```
+
+## `capabilities`
+
+Array of capabilities that the plugin supports.
+
+```yaml
+capabilities:
+- catalog
+- discover
+- state
+- about
+- stream-maps
+```
+
+## `pip_url`
+
+A string with the plugin's [`pip install`](https://pip.pypa.io/en/stable/cli/pip_install/#options) argument. Can point to multiple packages and include any of the pip install options.
+
+```yaml
+pip_url: apache-airflow==2.1.2 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.1.2/constraints-${MELTANO__PYTHON_VERSION}.txt
+```
+
+## `maintenance_status`
+
+The maintenance status of the plugin.
+
+```yaml
+maintenance_status: active
```

Comment:
Should we be listing the available options here? Link to JSON schema?

Reply:
Makes sense to do both 🌝